app/server/.eslintrc.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
         "no-unused-vars": "off",
         "@typescript-eslint/no-unused-vars": "error",
         "no-underscore-dangle": "off",
+        "max-classes-per-file": ["error", 2]
```

Comment:
You have changed this from default 1 class per file, I think it would make a case for readability if you left default 1 class per file. It's up to you, I am not really bothered.

Reply:
Yeah, fair enough!